jenkins/README.md

```diff
@@ -37,6 +39,8 @@ and deletion operations, there are separate triggers phrases as below:
 
 * **/keep-test-ubuntu-integration-main** run keep integration tests with CAPM3 API version v1beta1 and branch main on Ubuntu
 * **/keep-centos-integration-main** run keep integration tests with CAPM3 API version v1beta1 and branch main on CentOS
+* **/keep-ubuntu-integration-release-1-2** run keep integration tests with CAPM3 API version v1beta1 and branch release-1.2 on Ubuntu
+* **/keep-centos-integration-release-1-2** run keep integration tests with CAPM3 API version v1beta1 and branch release-1.2 on CentOS
```

Comment:
I am thinking if we should keep adding the branches here or just tell that release-1-x or release-x-x and tell the meaning of it.

Reply:
I would rather be explicit and moreover as we keep adding we would drop the support for older ones